iris-mpc-cpu/src/genesis/batch_generator.rs

```diff
@@ -207,12 +214,18 @@ impl fmt::Display for BatchSize {
 impl Batch {
     // Returns Iris serial id of batch's last element.
     pub fn id_end(&self) -> IrisSerialId {
-        self.vector_ids.last().map(|id| id.serial_id()).unwrap()
+        self.vector_ids
+            .last()
+            .map(|id| id.serial_id())
+            .unwrap_or_default()
```

Comment:
Could be empty in case of a Batch for which `iris_indexation_only` is true, i.e., a batch that only inserts iris codes but does not apply any mutation. This would happen in the case of a batch where all the items are deleted iris codes. We still want to copy those over to the HNSW schema